virtual/libusb/libusb-1-r2.ebuild

```diff
@@ -1,17 +1,12 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
-# $Id$
 
 EAPI=5
 inherit multilib-build
 
 DESCRIPTION="Virtual for libusb"
-HOMEPAGE=""
-SRC_URI=""
-
-LICENSE=""
 SLOT="1"
-KEYWORDS="~alpha amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc x86 ~amd64-fbsd ~x86-fbsd ~x86-freebsd ~amd64-linux ~arm-linux ~ia64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+KEYWORDS="~alpha amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
```

Comment:
It probably doesn't matter as much for virtual packages, but this version is out of sync between amd64 and arm64.

Reply:
It matters even less since it's only pulled in for the sdk (USE=usb)

Reply:
Looks like it's used by usbutils in prod images, but yes, it's not an important difference.